src/app/pages/dashboard/contagion-areas/contagion-areas.component.html

```diff
@@ -0,0 +1,20 @@
+<nb-card>
+  <nb-card-header>איזורי התפשטות</nb-card-header>
+  <nb-card-body>
+    <table [nbTreeGrid]="dataSource" nbSort (sort)="changeSort($event)">
+
+      <tr nbTreeGridHeaderRow *nbTreeGridHeaderRowDef="allColumns"></tr>
+      <tr nbTreeGridRow *nbTreeGridRowDef="let row; columns: allColumns"></tr>
+
+      <ng-container *ngFor="let column of allColumns" [nbTreeGridColumnDef]="column">
+        <th nbTreeGridHeaderCell [nbSortHeader]="getDirection(column)" *nbTreeGridHeaderCellDef>
+          {{columnsNameValueMap[column]}}
+        </th>
+        <td nbTreeGridCell *nbTreeGridCellDef="let row" [ngClass]="{'bold-text': column === 'location'}">
```

Comment:
Use `[class.bold-text]` syntax

Reply:
Maybe rename the class to `"important-column"`. This describes the reason and not the visuality